core/src/main/java/com/ashelkov/wallet/Utils.java

```diff
@@ -31,4 +32,35 @@ public static SecureRandom getSecureRandomInstance() {
         // Default to basic PRNG
         return new SecureRandom();
     }
+
+    /**
+     * Confirms passphrase by prompting user to re-enter and comparing hashes
+     *
+     * @param passphrase passphrase from the first prompt
+     * @return whether or not passphrases match
+     */
+    public static boolean confirmPasswordsMatch(String passphrase) {
+      String confirmation = getInputFromUser();
+
+      return passphrase.equals(confirmation);
+    }
+ 
+    /**
+     * Prompts user for passphrase confirmation via console
+     *
+     * @return passphrase confirmation from console
+     */
+    private static String getInputFromUser() {
+      Console cons;
+      char[] passwd;
+      String output = "";
+      // See: https://docs.oracle.com/javase/7/docs/api/java/io/Console.html#readPassword%28%29
+      if ((cons = System.console()) != null &&
```

Comment:
Under what circumstances could the console or reading the password from the console fail? May want to throw an exception (`PasswordValidator` could catch it, and it would be another fail condition).